dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m

```diff
@@ -100,6 +100,9 @@ - (void)waitForAndTapElement:(XCUIElement *)element {
     NSPredicate *hittable = [NSPredicate predicateWithFormat:@"exists == YES AND hittable == YES"];
     [self expectationForPredicate:hittable evaluatedWithObject:element handler:nil];
     [self waitForExpectationsWithTimeout:30.0 handler:nil];
+    // Sometimes, the element doesn't respond to the tap, it seems an XCUITest bug. Trying tap
+    // the app first to fix it.
+    [self.app tap];
```

Comment:
Does this tap on a random spot in the app?

Reply:
https://developer.apple.com/documentation/xctest/xcuielement/1618666-tap?language=objc "a hittable point", so I guess a random spot. Are you worried about this hitting other elements. I think the XCUITest framework should somehow find a spot that is not covered by other elements.

Reply:
I'm suspicious that this won't cause problems and tap other elements, but let's try it, I don't have any better ideas.